packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -375,6 +375,10 @@ export class EdgelessPageBlockComponent
         const element = this.surface.pickById(id);
         assertExists(element);
         if (element instanceof ConnectorElement) {
+          if (!this.connector.hasRelatedElement(element)) {
+            return;
+          }
```

Comment:
If just return, we need at some moment to update the path.

Reply:
connector will not show if we delete elements connectored to connector then undo

Reply:
The note element added afterward will trigger a connector path update.  Not sure if there's another scenario.

Reply:
https://github.com/toeverything/blocksuite/assets/58546692/c4109cc2-2eff-441f-b608-de9e93279a80

I think connector path update may better not trigged by other element.

Reply:
It's kind of like calling `connector.syncConnectorPos([newElement])` when new element is added.

Reply:
alright